assets/js/shortcut-key.js

```diff
@@ -0,0 +1,12 @@
+// Check if the user is on a Mac and update the shortcut key for search accordingly
+
+var isMac = navigator.platform.toUpperCase().indexOf('MAC') >= 0;
```

Comment:
Maybe we should hange this line to not pollute the global `window` scope?
E.g. by moving the whole code into a function that is automatically called?

Reply:
Updated. Let me know if you had something different in mind @CheariX